Svc/DpCatalog/DpCatalog.cpp

```diff
@@ -147,6 +149,8 @@ namespace Svc {
         this->m_dpTree = nullptr;
         // reset number of records
         this->m_numDpRecords = 0;
+        // reset number of files
+        
```

Comment:
There is a comment for resetting number of files, but no code to do it.  Is this comment wrong, or was something omitted?

Reply:
That was a stray comment from something I realized I didn't need to do. I'll push a delete of the line.